applications/ShapeOptimizationApplication/python_scripts/model_part_controller_factory.py

```diff
@@ -76,6 +77,7 @@ def __init__(self, model_settings, model):
         self.design_surface = None
         self.damping_utility = None
         self.direction_dampings = []
+        self.is_iteration_restart_files_written = self.model_settings["write_iteration_restart_files"].GetBool()
```

Comment:
```suggestion
        self.write_iteration_restart_files = self.model_settings["write_iteration_restart_files"].GetBool()
```

why not call it the same as the setting?